src/libsync/account.h

```diff
@@ -170,7 +171,7 @@ protected Q_SLOTS:
     QUrl _url;
     QList<QSslCertificate> _approvedCerts;
     QSslConfiguration _sslConfiguration;
-    QVariantMap _capabilities;
+    Capabilities *_capabilities;
```

Comment:
This should not be a pointer.  Just make it a normal object like the QVariantMap was.